source/extensions/common/wasm/wasm.cc

```diff
@@ -134,6 +134,8 @@ Wasm::Wasm(WasmHandleSharedPtr base_wasm_handle, Event::Dispatcher& dispatcher)
   ENVOY_LOG(debug, "Thread-Local Wasm created {} now active", active_wasms);
 }
 
+void Wasm::error(absl::string_view message) { ENVOY_LOG(trace, "Wasm VM failed {}", message); }
```

Comment:
This should be logged at a level higher than `trace`.

Reply:
Changed to "error".